### ISSUE TITLE
change the model.

### ISSUE THREAD

**Issue by shreyasudaya** [riya-singh28/TRINIT_dosAI_ML#21]
SVMs have more accuracy instead of RandomForestClassifier.
Instead of
```RF = RandomForestClassifier(n_estimators=50, max_features='sqrt', max_depth=32, criterion='gini')```
We can do
from sklearn.svm import SVC
and instead make the model an SVC and fit it to that. The accuracy of SVC is much better compared to RFC and has shorter computational time, compared to Random Forest which takes alot of time.

**Comment by shreyasudaya** [comment]
```from sklearn.svm import SVC```
```RF=SVC()```
